tests/core/editable/keystrokes/delbackspacequirks/manual/noselection.md

```diff
@@ -0,0 +1,14 @@
+@bender-tags: 4.7.0, tc, 13096
+@bender-ui: collapsed
+@bender-ckeditor-plugins: toolbar, wysiwygarea
+
+1. Open browser console.
```

Comment:
Hmmm.
I see that currently problem might be reproduced only on Chrome browser. FF, IE and Safari seems to don't have such glitch and caret remain in editor for the whole time.
I precise description to use Chrome browser for testing.

Does it help in this case or should I make some more detail description?